src/PhpSpreadsheet/Calculation/Calculation.php

```diff
@@ -3155,7 +3155,8 @@ private static function translateFormula(array $from, array $to, string $formula
                 $i = false;
                 foreach ($temp as &$value) {
                     //    Only adjust in alternating array entries
-                    if ($i = !$i) {
+                    $i = !$i;
+                    if ($i) {
```

Comment:
If we are going to change the instances of this, then using the more explicit `if ($i === true) {` would probably be better; and perhaps even renaming `$i` as something more meaningful like `$notWithinQuotes`

Reply:
Done.